tchannel-example/src/main/java/com/uber/tchannel/thrift/GetValueRequestHandler.java

```diff
@@ -31,11 +31,11 @@
 
 import java.util.Map;
 
-public class GetValueHandlerDefault extends DefaultRequestHandler<KeyValue.getValue_args, KeyValue.getValue_result> {
+public class GetValueRequestHandler extends DefaultRequestHandler<KeyValue.getValue_args, KeyValue.getValue_result> {
```

Comment:
ditto, about breaking the merge, we can move the renaming changes in a different PR ?